common/src/main/java/com/wynntils/core/managers/ManagerRegistry.java

```diff
@@ -72,6 +75,19 @@ public static void addDependency(Feature dependant, Class<? extends Model> depen
         updateManagerState(dependency);
     }
 
+    public static void addFunctionDependency(
```

Comment:
I assume this is only here because features don't implement the interface yet?

Reply:
Yes, if/when we unify the dependency interface between features and functions, this will go away (or maybe rather the current Feature variant goes away).